src/components/BuySell/BuySellWidget.tsx

```diff
@@ -1,26 +1,38 @@
-import React from 'react'
+import React, { ComponentProps } from 'react'
 import styled from 'styled-components'
 
 import BuySellSelector from './components/BuySellSelector'
 import TokenInputs from './components/TokenInputs'
 import OrderSummary from './components/OrderSummary'
 import BuySellButton from './components/BuySellButton'
 import HowToBuyLink from './components/HowToBuyLink'
+import ExchangeSelector from './components/ExchangeSelector'
 
-const BuySellWidget: React.FC = () => {
+interface Props {
+  setIsIssuance: any
```

Comment:
label this function type: i thought it could have been a boolean or a function
Let's also prefix our functions in react components with `on` 

```
interface Props {
   onSetIsIssuance: (...args: any[]) => any;
}
```
Feel free to be more specific on input/output args typing if you feel comfortable, but at least type it as a function

Reply:
fixed

Reply:
prefix your methods with `on`